TheWarWithin/RogueSubtlety.lua

```diff
@@ -370,12 +369,17 @@ spec:RegisterAuras( {
         max_stack = function() return combo_points.max end,
         copy = { "supercharge", "supercharged", "supercharger" }
     },
-    symbols_of_death_crit = {
+    symbols_of_death = {
+        id = 212283,
+        duration = 10,
+        max_stack = 1,
+    },
+    --[[symbols_of_death_crit = {
```

Comment:
Can delete if we're removing it.

Note that when this sort of removal is new (i.e., people may have a priority loaded t hat references an old aura), I'll often leave in an aura stub (or make a copy).  Not needed here, though.